tests/Integration/Database/EloquentWhereHasTest.php

```diff
@@ -26,7 +26,7 @@ protected function setUp(): void
         Schema::create('texts', function (Blueprint $table) {
             $table->increments('id');
             $table->unsignedInteger('post_id');
-            $table->boolean('content');
+            $table->text('content');
```

Comment:
`content` for these tests relies on string values to this column type was incorrect.